starlette/testclient.py

```diff
@@ -259,13 +266,15 @@ def __init__(
         self.headers.update({"user-agent": "testclient"})
         self.base_url = base_url
 
-    def request(self, method: str, url: str, **kwargs) -> requests.Response:
+    def request(  # type: ignore
+        self, method: str, url: str, **kwargs: typing.Any
+    ) -> requests.Response:
         url = urljoin(self.base_url, url)
         return super().request(method, url, **kwargs)
 
     def websocket_connect(
-        self, url: str, subprotocols=None, **kwargs
-    ) -> WebSocketTestSession:
+        self, url: str, subprotocols: Subprotocols = None, **kwargs: typing.Any
+    ) -> typing.Any:
```

Comment:
I'll have a think about how we can change this around so that we've got the correct `WebSocketTestSession` annotation. Leaving this comment as a reminder to myself.

Reply:
Is the `return` was missing, at line 289?

Reply:
No, it's just that the codepath *always* follows the except case.